src/updates/index.md

```diff
@@ -1,42 +1,174 @@
+
+#### Q1 2020 Profit and Loss
+
+We recorded a $723.95 loss in Q1 2020
+
+We received $512 in gross receipts, all of which is attributable to our maintenance efforts on [Moment Park](https://www.momentpark.com/)
+
+Our largest costs were:
+*   Puchasing a $600 team license for [TailwindUI](https://tailwindui.com/) to streamline design and delivery of web sites and applications without a dedicated designer or front-end wizard.
+*   Hiring a tax professional to prepare and file our 2019 taxes.
+
+[Download Zinc Collective's 2020 Q1 Profit and Loss statement](/financial-overview/2020-q1-profit-and-loss.pdf)
+
+#### Q1 2020 Balance Sheet
+
+100% of financial assets are held in an FDIC insured cash account.
+
+Zinc Collective carried no liabilities.
+
+[Download Zinc Collective's 2020 Q1 Balance Sheet](/financial-overview/2020-q1-balance-sheet.pdf)
+
+All representations of information are made as of March 31st, 2020
+
+### 2019 Annual Report, January 1st through December 31st
+
+#### End of 2019 Profit and Loss
+
+[Download Zinc Collective LLC's 2019 Profit and Loss statement](/financial-overview/2019-profit-and-loss.pdf)
+
+#### End of 2019 Balance Sheet
+
+[Download Zinc Collective LLC's 2019 Balance Sheet](/financial-overview/2019-balance-sheet.pdf)
+
+These mirror the [Q4 2019 Profit and Loss](#2019-q4-profit-and-loss) and [Q4 2019 Balance Sheet](#2019-q4-balance-sheet) because we formally began operations on September 1st, 2019
+
+### Q4 2019, October 1st through December 31st
+
+#### Q4 2019 Profit and Loss
+
+We received $1,281.60, all of which is attributable to our maintenance efforts on [Moment Park](https://www.momentpark.com/)
+
+Our largest costs were paying our $800 per year minimum California Franchise Tax exercised against Limited Liability Companies.
+
+[Download Zinc Collective's 2019 Q4 Profit and Loss statement](/financial-overview/2019-q4-profit-and-loss.pdf)
+
+Efforts will be made to map revenues to projects on the profit and loss statements going forward.
+
+####   Q4 2019 Balance Sheet
+
+100% of financial assets are held in an FDIC insured cash account.
+
+Zinc Collective carried no liabilities.
+
+[Download Zinc Collective's 2019 Q4 Balance Sheet](/financial-overview/2019-q4-balance-sheet.pdf)
+
+All representations of information are made as of December 31st, 2019
+
+</details>
+
+
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6 float-left mr-2 mt-8">
+  <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 14.25v-2.625a3.375 3.375 0 00-3.375-3.375h-1.5A1.125 1.125 0 0113.5 7.125v-1.5a3.375 3.375 0 00-3.375-3.375H8.25m0 12.75h7.5m-7.5 3H12M10.5 2.25H5.625c-.621 0-1.125.504-1.125 1.125v17.25c0 .621.504 1.125 1.125 1.125h12.75c.621 0 1.125-.504 1.125-1.125V11.25a9 9 0 00-9-9z" />
+</svg>
+
+
+## Meeting Notes
+
 We currently have a monthly members-only meeting. We will publish meeting summaries here.
```

Comment:
I think it's fine to delete the meeting notes section for now.

Reply:
(My reasoning for this is because we haven't been keeping them uip to date... We could instead link ot the open collective community updates)

Reply:
added open collective community updates link above finance info in https://github.com/zinc-collective/www.zinc.coop/pull/107/commits/1c028a662ae0d549c889a5381da3f09d94d96177